client/src/app/shared/components/os-sort-filter-bar/os-sort-filter-bar.component.ts

```diff
@@ -80,7 +81,7 @@ export class OsSortFilterBarComponent<V extends BaseViewModel> {
      */
     public get displayedCount(): number {
         if (this.filterCount === undefined || this.filterCount === null) {
-            return this.filterService.filterCount;
+            return this.filterService.filteredCount;
         } else {
```

Comment:
*should* be correct (my IDE agrees), but I'm not yet sure if this borked something